editor/ramatak/ramatak_settings_editor.cpp

```diff
@@ -186,11 +193,22 @@ void RamatakSettingsAdUnitSetEditor::_remove_ad_unit() {
 	Dictionary ad_units_config = ProjectSettings::get_singleton()->get("ramatak/monetization/ad_units");
 	Vector<int> selected_items = edit_items_list->get_selected_items();
 	if (selected_items.size() > 0) {
+		int item_to_remove = selected_items.get(0);
 		// DEV_ASSERT(ad_unit_editor->get_ad_unit() != "");
 		ad_units_config.erase(ad_unit_editor->get_ad_unit());
-		edit_items_list->remove_item(selected_items.get(0));
-		ad_unit_editor->clear();
-		ad_unit_editor->set_disabled(true);
```

Comment:
Since I'm not super-familiar with this part of the codebase yet, I may be making some dumb comments. This line isn't still needed, right? I don't see it below anywhere, wasn't sure if it was just missing or intentionally entirely deleted.

Reply:
Yeah, I'm not entirely sure it was necessary earlier either though because the editor pane is being cleared so it would be hard to interact with it anyway.